src/app/ErrorView/ErrorView.tsx

```diff
@@ -44,15 +44,31 @@ export interface ErrorViewProps {
 }
 
 export const ErrorView: React.FunctionComponent<ErrorViewProps> = (props) => {
-    return (<>
-        <br/>
-        <Bullseye>
-            <ExclamationCircleIcon size='md' color='Red'/>
-        </Bullseye>
-        <Bullseye>
-            <Text>
+
+    if (props.message === 'Target SSL Untrusted'){
+        return (<>
+            <br/>
+            <Bullseye>
+                <ExclamationCircleIcon size='md' color='Red'/>
+            </Bullseye>
+            <Bullseye>
+                <Text>Error:&nbsp;{props.message}</Text>
+            </Bullseye>
+            <Bullseye>
+                <Text>
+                    To add the SSL certificate for this target, click <a href='/Security'>here</a> or go to the Security tab
+                </Text>
+            </Bullseye>
+        </>)
+    } else {
```

Comment:
Wherever this specialized case of an error message is implemented, whether it's here in the `ErrorView` or as a dialog, it shouldn't be done with a simple `if/else` check on some property. Otherwise, the view/dialog will balloon in size as more cases are added. This would be a good time to exercise some React view composition.